packages/core/core/src/domain/module/index.ts

```diff
@@ -103,8 +103,8 @@ export const createModule = (namespace: string, rawModule: RawModule, strapi: St
     get routes() {
       return rawModule.routes ?? {};
     },
-    config(path: string, defaultValue: unknown) {
-      return strapi.get('config').get(`${namespace}.${path}`, defaultValue);
+    config(path: PropertyPath, defaultValue: unknown) {
+      return strapi.get('config').get([namespace, path], defaultValue);
```

Comment:
does it accept nested arrays ? should we concat ?

Reply:
Gah, you were too quick! I already fixed it with flatten